library/Rules/Length.php

```diff
@@ -15,13 +15,33 @@
 
 use Respect\Validation\Exceptions\ComponentException;
 
-class Length extends AbstractRule
+/**
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
```

Comment:
Would be nice to add a one-line documentation in this docblock (which can be found in documentation of the rule)